apps/website-25/src/pages/courses/[courseId]/unit/[unitId].tsx

```diff
@@ -0,0 +1,41 @@
+import { COURSE_UNITS } from '@bluedot/ui/src/constants';
+import { useRouter } from 'next/router';
+import useAxios from 'axios-hooks';
+import { ProgressDots } from '@bluedot/ui';
+import UnitLayout from '../../../../components/courses/UnitLayout';
+import { GetUnitResponse } from '../../../api/courses/[courseId]/[unitId]';
+import { ROUTES } from '../../../../lib/routes';
+
+const CourseUnitPage = () => {
+  const { query: { courseId, unitId } } = useRouter();
+
+  const [{ data, loading }] = useAxios<GetUnitResponse>({
+    method: 'get',
+    url: `/api/courses/${courseId}/${unitId}`,
+  });
+
+  const unitNumber = unitId ? parseInt(unitId as string) : 0;
```

Comment:
Seems slightly odd to fallback to 0?

Maybe should display an error page instead?